x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -1732,7 +1732,7 @@ public enum Cap {
         /**
          * Returns the top snippets for given text content and associated query.
          */
-        TOP_SNIPPETS_FUNCTION(Build.current().isSnapshot()),
+        TOP_SNIPPETS_FUNCTION(),
```

Comment:
Should this still have the parentheses here? (i.e. should this just be `TOP_SNIPPETS_FUNCTION,` ?)